syncstorage/storage/sql/dbconnect.py

```diff
@@ -102,6 +102,22 @@ def __new__(cls, *args, **kwds):
 )
 
 
+# Table listing migration state for users
+#
+# This table contains the users in progress for migration. This table may
+# be updated by the external user migration script
+# A UID appearing in this table should cause the server to return a 5xx
```

Comment:
Nit: since most instances of "uid" or "userid" in this codebase refer to the numeric id allocated by tokenserver, it may be worth mentioning explicitly that this is the fxa uid in the comment here (which you do also mention by explicit name in the table declaration below 👍)